content/efcore/getting-started/samples/loading-related-data-tutorial/count-method/README.md

```diff
@@ -0,0 +1,10 @@
+# Count Method
+
+This code example demonstrates how to use EF Core to query related data from the database using the .Count method.
+
+## Code Files:
+- [Program.cs](Program.cs)
+- [LibraryContext.cs](../../shared/LibraryContext.cs)
+- [Book.cs](../../shared/Book.cs)
+- [Author.cs](../../shared/Author.cs)
+- [Edition.cs](../../shared/Edition.cs)
```

Comment:
I'm definitely fine with that, but don't know if that would involve changes to the work already done on the markdown engine, and we don't really have time to make those changes